eraser-map.yaml

```diff
@@ -861,7 +861,6 @@ styles:
     icons:
         base: points
         texture: pois
-        interactive: true
```

Comment:
In case this clarifies at all (maybe not :), everything in `style` affects *GL render state*, nothing technically "inherits" to the draw group. Whereas draw groups nested in sub-layers inherit properties from those above them. So properly speaking if we made a connection here it would probably make more sense to think of it as a default value than an inherited one.